control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1886,8 +1886,8 @@ func reconcileCreationBlockerWebhook(wh *admissionregistrationv1.ValidatingWebho
 				Service: &admissionregistrationv1.ServiceReference{
 					Namespace: "default",
 					Name:      "xxx-invalid-service-xxx",
-					Path:      pointer.String("/validate"),
-					Port:      pointer.Int32(443),
+					Path:      ptr.To("/validate"),
+					Port:      ptr.To(int32(443)),
```

Comment:
you can specify the generic type directly, no need for a cast
```suggestion
					Port:       ptr.To[int32](443),
```